public/tenants/planet/manifest.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "Plant for the Planet",
+    "short_name": "Plant",
+    "start_url": "/?utm_source=pwa",
+    "description": "Plant trees around the world - Plant-for-the-Planet",
+    "display": "fullscreen",
```

Comment:
`fullscreen` looks best, but with rounded corners and notch, the back arrow for "tree registrations" or "profile editing" is not clickable.